cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -187,6 +188,45 @@ def test_cluster_name_validator_slurm_accounting(cluster_name, scheduling, shoul
     assert_failure_messages(actual_failures, expected_message)
 
 
+@pytest.mark.parametrize(
+    "description, custom_settings, deny_list, settings_level, expected_message",
+    [
+        (
+            "No error when custom settings are not in the deny_list",
+            {"Allowed1": "Value1", "Allowed2": "Value2"},
+            ["Denied1", "Denied2"],
+            SlurmCustomSettingLevel.QUEUE,
+            "",
+        ),
+        (
+            "Fails when custom settings are in the deny_list, invalid parameters are reported",
+            {"Denied1": "Value1", "Denied2": "Value2"},
+            ["Denied1", "Denied2"],
+            SlurmCustomSettingLevel.QUEUE,
+            "Setting or overriding the following custom Slurm settings at Queue level is not allowed: Denied1,Denied2",
+        ),
+        (
+            "No error when custom settings are not in the deny_list",
+            {"Allowed1": "Value1", "Allowed2": "Value2"},
+            ["Denied1", "Denied2"],
+            SlurmCustomSettingLevel.COMPUTE_RESOURCE,
+            "",
+        ),
+        (
+            "Fails when custom settings are in the deny_list, invalid parameters are reported",
+            {"Denied1": "Value1", "Denied2": "Value2"},
+            ["Denied1", "Denied2"],
+            SlurmCustomSettingLevel.COMPUTE_RESOURCE,
+            "Setting or overriding the following custom Slurm settings at"
+            " ComputeResource level is not allowed: Denied1,Denied2",
```

Comment:
For better clarity, please put the full sentence here.

Reply:
Not sure I got your point.
It is the full sentence, I just had to split it in two lines to make the code-linter happy

Reply:
With the rewording fits into one line, so it should be easier to check that it is the full sentence.